src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -2852,26 +2854,40 @@ namespace AirflowNetworkBalanceManager {
         CurrentModuleObject = "AirflowNetwork:MultiZone:Surface";
         for (int i = 1; i <= AirflowNetworkNumOfSurfaces; ++i) {
             if (MultizoneSurfaceData(i).SurfNum == 0) continue;
-            bool has_DOP_or_SOP{false};
+            bool has_Opening{false};
             // This is terrible, should not do it this way
             auto afe = solver.elements.find(MultizoneSurfaceData(i).OpeningName);
             if (afe != solver.elements.end()) {
                 auto type = afe->second->type();
-                has_DOP_or_SOP = (type == ComponentType::DOP) || (type == ComponentType::SOP);
+                has_Opening = (type == ComponentType::DOP) || (type == ComponentType::SOP) || (type == ComponentType::HOP);
```

Comment:
Fix bug that ignored processing the surface-level Ventilation Availability Schedule for horizontal openings.